### ISSUE TITLE
a11y hit Test support in flutter

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#47386]
https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=36664. 

/cc @dnfield --- Dan, can you take a look at this, please?

**Comment by dnfield** [comment]
Working on this actively.

**Comment by dnfield** [comment]
https://github.com/flutter/engine/pull/14602 is supposed to help this.

I'm blocked right now because the a11y subsystem doesn't seem to call the HitTest method when expected. Working on that with some Fuchsia a11y folks.

**Comment by dnfield** [comment]
Blocked on https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=43405 - hits aren't delivered to flutter mods when the screen reader is enabled.

**Comment by dnfield** [comment]
Shifting milestone to near term goals while blocker is investigated.

**Comment by dnfield** [comment]
Unblocked, PR open and should be landed soon.
